cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -181,19 +181,33 @@ def __init__(self, scope: core.Construct, construct_id: str, imagebuild: ImageBu
         ami_distribution_configuration = {
             "Name": self._set_ami_name(),
             "AmiTags": ami_tags,
+            "LaunchPermissionConfiguration": dev_settings.distribution_configuration.launch_permission
+            if dev_settings and dev_settings.distribution_configuration
+            else None,
         }
 
+        distributions = [
+            imagebuilder.CfnDistributionConfiguration.DistributionProperty(
+                ami_distribution_configuration=ami_distribution_configuration,
+                region=core.Fn.sub("${AWS::Region}"),
+            )
+        ]
+
+        if dev_settings and dev_settings.distribution_configuration and dev_settings.distribution_configuration.regions:
+            for region in set(map(str.strip, dev_settings.distribution_configuration.regions.split(","))):
+                distributions.append(
+                    imagebuilder.CfnDistributionConfiguration.DistributionProperty(
+                        ami_distribution_configuration=ami_distribution_configuration,
+                        region=region,
```

Comment:
Will it cause an issue if regions contains ${AWS::Region} region? I think we can replace ${AWS::Region} here with region from cli, and add a property **region** in imagebuilder config, like
```
distributions = [            
          imagebuilder.CfnDistributionConfiguration.DistributionProperty(
                ami_distribution_configuration=ami_distribution_configuration,
                region=imagebuild.region,
            )
]

if dev_settings and dev_settings.distribution_configuration and dev_settings.distribution_configuration.regions:
    region_set = set(map(str.strip, dev_settings.distribution_configuration.regions.split(",")))
    if imagebuild.region in region_set:
       region_set.remove(imagebuild.region)
    for region in region_set:
        distributions.append(
            imagebuilder.CfnDistributionConfiguration.DistributionProperty(
                ami_distribution_configuration=ami_distribution_configuration,
                region=region,
           )
     )
```

Reply:
done